easybuild/easyconfigs/m/magma/magma-1.6.2-CrayGNU-5.2.40.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = "ConfigureMake"
+
+name = 'magma'
+version = '1.6.2'
+
+homepage = 'http://icl.cs.utk.edu/magma/'
+description = """The MAGMA project aims to develop a dense linear algebra library similar to LAPACK 
+    but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems."""
+
+toolchain = {'name': 'CrayGNU', 'version': '5.2.40'}
+toolchainopts = {'pic': True, 'dynamic': True,  'openmp': True, }
+
+preinstallopts = "export EBINSTALLPREFIX=%(installdir)s && "
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://icl.cs.utk.edu/projectsfiles/magma/downloads/']
+
+patches = [('make.inc', 'magma-1.6.2/')]
```

Comment:
i've tried that, but patching fail as it seems to expect the dest directory as the second argument: Can't patch directory /dev/shm/perettig/magma/1.6.2/CrayGNU-5.2.40/magma-1.6.2/make.inc: no such directory")

Should I open an issue for that?